files-community/Files
# Feature: Added support for choosing the dual pane orientation

### Resolved / Related Issues

- Closes #12427

### Steps used to test these changes

- Check for splitting
  1. Click the Tab Actions button
  2. Click 'Split pane'
  3. Add pane in both ways
- Check for arranging
  1. Click the Tab Actions button
  2. Click 'Arrange pane'
  3. Arrange panes in both ways
- Check for the saved arrangement of panes
  1. Arrange panes vertically
  2. Close the window
  3. Re-open
- Check for the setting to open dual pane always
  1. Select Horizontal/Vertical
  2. Open a new tab
  3. Open a new pane
